### ISSUE TITLE
Pre-work Checklist: Developer: Niva Razin

### ISSUE THREAD

**Issue by nnr-nnr** [hackforla/website#4063]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by nnr-nnr** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
~6 hours

**Comment by nnr-nnr** [comment]
Updates as of 3/21
- Have attended all Tuesday Developer meetings and Thursday Developer OHs this month (March 2023)
- Reviewed 4 PRs (#4184, #4222, #4223, #4224)
- * I have some questions about #4223 — there seem to be some typos/unclear instructions in the corresponding Issue #4010
- Submitted 2 PRs (#4201, #3878), both `good first issue`'s per Bonnie's instruction, as there aren't currently any `good second issue`'s
- Hoping to complete a `small` and maybe `medium` issue by next Tuesday, as well as more PR reviews

**Comment by nnr-nnr** [comment]
1. Progress: Submitted a PR for a `Complexity: small` issue (#4235/#4265), which has been merged. Attended Sunday all hands meeting to get a `Complexity: medium` issue (#2249), which I'm currently working on. Completed 4 PR reviews (#4223, #4270, #4274, #4234). Found a bug in #4270, and am creating an emergent request issue for it.
2. Blockers: N/A (got questions answered at Thurs. OHs)
3. Availability: Sat–Sun 12–3 PM PST, M–F 3–8 PM PST
4. ETA: in a couple of weeks, once I finish the medium issue and pick up, complete a large issue.
5. Pictures: N/A

**Comment by nnr-nnr** [comment]
Continuing to work on the `medium`-sized issue: https://github.com/hackforla/website/issues/2249

**Comment by roslynwythe** [comment]
Hi @nnr-nnr please leave an update at least every two weeks, using the guidelines from the above comment.  Don't forget to remove the '2 weeks inactive' label.  Thanks

**Comment by roslynwythe** [comment]
Hi @nnr-nnr.  Please create an update as described in the above comment, and then don't forget to remove the `2 weeks inactive` label.   If you are not currently working on other issues, then we would like to see updates at least every two weeks, to inform us of your progress and any problems you might be experiencing, and to let us know that you are still interested in participating in Hack for LA.  

If we do not hear from you by 7/24/2023, we will assume that you are not interested in participating in Hack for LA at this time, and we will close this issue, but you can reopen it at a later date, and move it into the "In Progress" column.  Thank you.

**Comment by nnr-nnr** [comment]
1. Progress: working on a medium complexity [issue](https://github.com/hackforla/website/issues/4867)
2. Blockers: NA
3. Availability: see linked issue
4. ETA: 9/30
5. Pictures:NA

**Comment by ExperimentsInHonesty** [comment]
@nnr-nnr In case you were waiting on a [Medium size front end issue, there are two available now, on the project board, prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-7198257)

**Comment by HackforLABot** [comment]
## Activity Log: nnr-nnr
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- nnr-nnr opened: https://github.com/hackforla/website/issues/4063 at 02/28/2023, 08:03 PM PST
- nnr-nnr assigned: https://github.com/hackforla/website/issues/4063 at 02/28/2023, 08:08 PM PST
- nnr-nnr assigned: https://github.com/hackforla/website/issues/3997 at 03/14/2023, 06:11 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4063#issuecomment-1469497271 at 03/15/2023, 12:38 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/3997#issuecomment-1469500011 at 03/15/2023, 12:41 AM PDT
- nnr-nnr opened: https://github.com/hackforla/website/pull/4201 at 03/17/2023, 01:11 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4184#issuecomment-1473378161 at 03/17/2023, 01:24 AM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4184#pullrequestreview-1346835489 at 03/17/2023, 04:44 PM PDT
- nnr-nnr PR was merged: https://github.com/hackforla/website/pull/4201#event-8786877846 at 03/18/2023, 09:04 PM PDT
- nnr-nnr issue closed by PR 4201: https://github.com/hackforla/website/issues/3997#event-8786877883 at 03/18/2023, 09:04 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4224#issuecomment-1478521037 at 03/21/2023, 01:12 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4224#pullrequestreview-1351388369 at 03/21/2023, 02:10 PM PDT
- nnr-nnr assigned: https://github.com/hackforla/website/issues/3878 at 03/21/2023, 02:15 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/3878#issuecomment-1478599795 at 03/21/2023, 02:25 PM PDT
- nnr-nnr opened: https://github.com/hackforla/website/pull/4228 at 03/21/2023, 04:32 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4222#issuecomment-1478729403 at 03/21/2023, 04:38 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4222#pullrequestreview-1351580167 at 03/21/2023, 06:07 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4223#issuecomment-1478790895 at 03/21/2023, 06:18 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4223#pullrequestreview-1351713337 at 03/21/2023, 09:51 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4063#issuecomment-1478923014 at 03/21/2023, 10:06 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4234#issuecomment-1478927131 at 03/21/2023, 10:12 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4234#pullrequestreview-1353457902 at 03/22/2023, 01:51 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4223#pullrequestreview-1353731944 at 03/22/2023, 07:16 PM PDT
- nnr-nnr PR was merged: https://github.com/hackforla/website/pull/4228#event-8832447458 at 03/23/2023, 02:03 PM PDT
- nnr-nnr issue closed by PR 4228: https://github.com/hackforla/website/issues/3878#event-8832447800 at 03/23/2023, 02:03 PM PDT
- nnr-nnr assigned: https://github.com/hackforla/website/issues/4235 at 03/23/2023, 02:26 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4235#issuecomment-1481945824 at 03/23/2023, 02:36 PM PDT
- nnr-nnr opened: https://github.com/hackforla/website/pull/4265 at 03/23/2023, 04:01 PM PDT
- nnr-nnr PR was merged: https://github.com/hackforla/website/pull/4265#event-8842509391 at 03/24/2023, 01:28 PM PDT
- nnr-nnr issue closed by PR 4265: https://github.com/hackforla/website/issues/4235#event-8842509619 at 03/24/2023, 01:28 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4223#pullrequestreview-1357923934 at 03/25/2023, 06:19 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4274#issuecomment-1483966636 at 03/25/2023, 06:32 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4270#issuecomment-1484163925 at 03/26/2023, 10:23 AM PDT
- nnr-nnr assigned: https://github.com/hackforla/website/issues/2249#issuecomment-1482219221 at 03/26/2023, 11:18 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1484196141 at 03/26/2023, 12:17 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4270#pullrequestreview-1358111274 at 03/26/2023, 01:30 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4274#pullrequestreview-1358164665 at 03/26/2023, 05:06 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4270#issuecomment-1484285589 at 03/26/2023, 05:10 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4274#issuecomment-1491136438 at 03/30/2023, 05:36 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4063#issuecomment-1492453676 at 03/31/2023, 11:59 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1492508865 at 03/31/2023, 12:44 PM PDT
- nnr-nnr opened: https://github.com/hackforla/website/issues/4389 at 04/03/2023, 03:01 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1500543036 at 04/07/2023, 11:47 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1500558240 at 04/07/2023, 12:06 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4437#issuecomment-1501174883 at 04/09/2023, 10:21 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4444#issuecomment-1501175092 at 04/09/2023, 10:22 AM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4444#pullrequestreview-1379936221 at 04/11/2023, 12:27 PM PDT
- nnr-nnr opened: https://github.com/hackforla/website/issues/4484 at 04/11/2023, 08:05 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/4437#pullrequestreview-1381841679 at 04/12/2023, 11:01 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4063#issuecomment-1522419157 at 04/25/2023, 02:08 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1522664094 at 04/25/2023, 07:14 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1553775073 at 05/18/2023, 04:22 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4708#issuecomment-1560246073 at 05/23/2023, 04:15 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/4708#issuecomment-1561839627 at 05/24/2023, 12:49 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1604333206 at 06/23/2023, 07:04 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1650813088 at 07/25/2023, 06:07 PM PDT
- nnr-nnr unassigned: https://github.com/hackforla/website/issues/2249#issuecomment-1650813088 at 07/25/2023, 06:07 PM PDT
- nnr-nnr assigned: https://github.com/hackforla/website/issues/4867 at 07/25/2023, 07:22 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/5058#issuecomment-1650878591 at 07/25/2023, 07:28 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/pull/5054#issuecomment-1650879675 at 07/25/2023, 07:30 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/5054#pullrequestreview-1548861931 at 07/26/2023, 06:49 PM PDT
- nnr-nnr submitted review: https://github.com/hackforla/website/pull/5058#pullrequestreview-1548873368 at 07/26/2023, 07:01 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4867#issuecomment-1652805044 at 07/26/2023, 07:03 PM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4867#issuecomment-1666019015 at 08/04/2023, 11:28 AM PDT
- nnr-nnr commented on: https://github.com/hackforla/website/issues/4063#issuecomment-1666023526 at 08/04/2023, 11:31 AM PDT
- nnr-nnr unassigned: https://github.com/hackforla/website/issues/4867 at 09/19/2023, 07:08 PM PDT
